concepts/observability-strategy/assets/strategy-future.drawio.svg

```diff
@@ -0,0 +1,620 @@
144e" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="1041px" height="851px" viewBox="-0.5 -0.5 1041 851" content="&lt;mxfile&gt;&lt;diagram id=&quot;scDzJU2_el85Nsdz-7Bu&quot; name=&quot;Page-1&quot;&gt;7Vxbc+I2FP41zLQPMJblG48EyHa72W6m7E67jwILcGMsKpsE+usr27KxJTkhQTYtWZjZ4GNJtr9z0XeO5O3B8Wb/gaLt+jPxcdgzDX/fg5OeaQLPNNifVHIoJK6TS1Y08LnsKJgF/2Au5B1Xu8DHca1hQkiYBNu6cEGiCC+SmgxRSp7qzZYkrF91i1ZYEswWKJSlfwR+ss6lnuke5b/gYLUurgycYX5mg4rG/EniNfLJU0UEpz04poQk+a/NfozDFL0Cl7zfbcPZ8sYojpJTOjhe3uMRhTv+cPzGkkPxtJTsIh+nHYwevHlaBwmebdEiPfvEFMxk62QTsiPAfi6DMByTkNCsL7zNPkweJ5Q84MoZJ/ukPUiUVOT5h8nlR+FP94hpgvcVEX+0D5hscEIPrAk/2/c4zNzQ+mDIBU9HtblFo3VFZZ7BhYibyqoc/Igm+8EBbQB3qBlcH8XrrK0C6Yk99SaWCmkj+2hC1BYRtWRELaBA1LI1IOoaHSI6ddJv14hCU2GipgJQ09MBKHgZ0AbvjkiENUFgiUYFZAyAo8DA0YKBKWEw2m7DYIGSgEQdhUMTWpbty+FQq6kBc2DXjc1TGJsCZy2+CyWYPx02iElmhzjBm46ALgFtFWgw8Nxh9VOD3bZl8zbgwLUVFq4DeasJ+XG4Y9BTCXr2nEkd3zqO3PcV4QCFwSpihwuGHhsY3qSoMVcKR/zEJvD99DJKhdZV3qaGHDHkuIp5TBV1dejDVuqjX/iB8Rva4DhD5t0pBgxtQTEKggEchado0YwjaeZbnPrHe1aJbddVYrvWQBHAVKRPi05cCXLss3SMHxKarMmKRCicHqUCOMc2d4RsuZb+wkly4Lkl2iWkrkO8D5I/0+7sUfOj75Uzkz0fOTs4FAcRe7RKp/Twe/XcsVt2VPQT5q45Wng+fJYkNGo1Jju64Ci5PLdLEF3hMq/JZSmCz+qe4pDRn8d6wqtSI+96TwJ2K0c3lqiGVVhRMUh+W7yfYA7ljZxmIXIS+zFm3K1nOmHqmnNaMx/n712aXmcu048zAxixBt52nyFbnGa/Vunfe0r2hzPJCbsStzTvTZlbm97tnsAJYVucUM6Q3wv1thQksC3q7clp8w8HebODqDTXloN4cn7+aTfHNMIJjq8zdTIGjtmcOvVVdNAAA7Ol3MmTiwOcEX5GEVoxjE3jd8ynXa1qsNPv21jAGRVSU4mvqp6nA9zGkkDL4Po29nxlRPHMOVSVpXXaOLwo5paEZBd0uqTGNWJ85MmnUWMMmObcZ2NTfqZYlrEqVL4g75UrN1L5Ri3XCHax9lQl2DyD1EywR5SiQ6XBNqXNcWVkkX8LNlbY3O3b2rMf+R009DbEaorA9XPUJK4vDWSa9am2TCr1Jw3AlLOGO7KKq5wo5zdfKYsqCvlndrFgESvd6Q7NcVh3gdOzfYoZ6ULzbLzUGLmy2eD2Tc+eKM2zcGspCpWroHzAXnWhURWdjEGqh7zrmVlg362bBTTrI5DlMsbnKtKTq2kfIxYFdhsGjjqPeEPJhoWYm+yruW6jYw53BqZTB1qxzDksE5DqjAJ1zChy0UxCvPU1JCAt9Xa6huS5EgbsGUPmdtdvfxKfUVifCngttnfC/oXWbQ96Ys1aYXu2cnVHi+2pthnkk1W8RVExXc0SQtN9LWn4d9AmNYX8XyYZMS53iINykmOXrHa9cgO2TzBgoMrwdVjw8Acf18XHi4moyscB4N5xYUIu2thLhPyF9i8RcqF3ue/ptYS87Ng0kD5CPpTD2D0ljHnHKvJdkHKNRYEpmNhTpTF3URQA4iaYTqsC5cCnoZ8nShqxn4ym3u34Utib4q62jrGXi7zPYN+Udp5Z8zXHF4MfOlAosXetAFNSwC37FSU3QSIr4Cc2T2QV4C9JtrGZE31Cf9aqEx9hb7lQ6cRZeHi+bNklKjX1Mn1Xr/u3VHkHhlwdFhHXCvhyuTQXSsB9Z+7YbccgGXDLPBFtWwfa8iaxVtFupo5Dx4Woe7SBYuNti8YtF01YzGfjrPFOEfRHIXu0TbYock1Gbwn00lIkzqrt+XpUINds7shDcEVB3PYuCa9cmvkVYd32e9kwYhkXBBjIrKVe93l5SweADXs6PlC0ZNHm1HpQO/sNXoxAlW0jQNMrQ6LLQNilRhXr4QEKzwxJLcTpblGRp8oikBjfPl5xNOkWZTlcf8Uh5gOJfOTLFlN0PiVs8uuGCKEBYzD0yvcsitKa5UkwA+CqlsusMgM6D2vdLwaKpM5Ov0rW0cFbl65QyQKOq8obVZYMTB3wFkT+SuGFQlrunAyujjhhKopVJGGPwyYqJp6N7tPKyDzG9BHNgzBI0lufsaPg3D1cOrBzxM2cfdhlSaMwxAp4OWDT6DGgJEq3LEggXf0LJq74MpYjz3vK9Tc9Bi2TrumewRVl9vxDMdVI7iojeXuqUdSkxMjCN0DLRdrgIV3rnqAE+WTFLjXbhrvogf0YDAbnVmpfyVtafYFcVJJpKJWkeplRT0yzJTDfywL2wABOr7qIbTkvvCHGDu4xDRjKqdufubRd+NiFl7ahBQUDfOXS9ivb923BPp9bCbca1tFfvRLuCmuE0sqTzq2pF3nlUYdrNBrzf237he28woigpe79aiMSbwOK06ROI5KTzP/x/ubSJ87f4Nw3WNwemjVFcH2eu9/Zqo/aL94eOmfDMzs8/n9QefPjf6sFp/8C&lt;/diagram&gt;&lt;/mxfile&gt;">
```

Comment:
Pls fix the spelling:

- `User Managed` => `User-Managed`
- `Kyma Managed`=> `Kyma-Managed`
- `FluentBit` => `Fluent Bit`
- `Storage & Analysis` => `Storage and Analysis`
- `Prometheus Alertmanager` => I believe this should be `Prometheus, Alertmanager` ?